mediacdn/sign_test.go

```diff
@@ -0,0 +1,148 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package snippets
+
+import (
+	"testing"
+	"time"
+)
+
+var testKey = []byte{34, 31, 185, 24, 168, 225, 242, 115, 112, 155, 38,
```

Comment:
Maybe add a comment for maintainers about how this value was produced?

Reply:
renamed to privateKey